helm-charts/templates/data-plane/gateway-components/gateway-runtime/enforcer-analytics-logs-volume-pvc.yaml

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2023, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+{{- if and .Values.wso2.apk.dp.gatewayRuntime.analytics .Values.wso2.apk.dp.gatewayRuntime.analytics.enabled .Values.wso2.apk.dp.gatewayRuntime.analytics.type }}
+{{- if eq .Values.wso2.apk.dp.gatewayRuntime.analytics.enabled true }}
+{{- if eq .Values.wso2.apk.dp.gatewayRuntime.analytics.type "File" }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: enforcer-analytics-logs-volume-pvc
+spec:
+  storageClassName: standard
```

Comment:
This should be configurable right?

Reply:
Resolved by https://github.com/wso2/apk/pull/1248/commits/5edad913d93dc631e4e6d354eecd36623fa365d0